src/main/resources/org/csanchez/jenkins/plugins/kubernetes/volumes/workspace/DynamicPVCWorkspaceVolume/help-accessModes.html

```diff
@@ -0,0 +1 @@
+A PersistentVolume can be mounted on a host in any way supported by the resource provider. As shown in the table below, providers will have different capabilities and each PV’s access modes are set to the specific modes supported by that particular volume. For example, NFS can support multiple read/write clients, but a specific NFS PV might be exported on the server as read-only. Each PV gets its own set of access modes describing that specific PV’s capabilities.
```

Comment:
Sounds like a copy paste from the kubernetes [PV](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) documentation. Should be reworded (it mentions a table that is not there).